--- a/quickassist/utilities/service/qat.service.in	2023-03-01 19:19:59.000000000 +0100
+++ b/quickassist/utilities/service/qat.service.in	2026-02-05 07:59:15.379502501 +0100
@@ -44,6 +44,8 @@ RuntimeDirectoryMode=755
 Environment="POLICY=0"
 EnvironmentFile=-/etc/sysconfig/qat
 ExecStartPre=/bin/sh -c "test $(getent group qat)"
+# A workaround for RHEL-133797 and RHEL-133799
+ExecStartPre=/bin/sh -c "restorecon /usr/lib/modules/$(uname -r)/modules.*"
 ExecStartPre=@prefix@/sbin/qat_init.sh
 ExecStart=@prefix@/sbin/qatmgr --policy=${POLICY}
 
